eden/graph.py

```diff
@@ -582,7 +582,8 @@ def _weight_preprocessing(self, graph):
                 if self.key_weight not in d:
                     graph.node[n][self.key_weight] = 1
 
-    def _edge_to_vertex_transform(self, original_graph):
+    @staticmethod
+    def _edge_to_vertex_transform(original_graph):
```

Comment:
Why do you need this to be static? If you need this function not bound to the class the function should probably not start with `_`, indication that its an inner helper function.